flink-yarn/src/main/java/org/apache/flink/yarn/cli/FlinkYarnSessionCli.java

```diff
@@ -478,12 +485,15 @@ public static void runInteractiveCli(YarnClusterClient yarnCluster, boolean read
 
 	public static void main(final String[] args) throws Exception {
 		final FlinkYarnSessionCli cli = new FlinkYarnSessionCli("", ""); // no prefix for the YARN session
-		Configuration flinkConfiguration = GlobalConfiguration.loadConfiguration();
+
+		final String configurationDirectory = CliFrontend.getConfigurationDirectoryFromEnv();
```

Comment:
Yes it should. The CLIs behave differently mainly because of historical reasons (different time when they were developed).